e2e/cypress/tests/integration/messaging/autocomplete_spec.js

```diff
@@ -319,29 +319,4 @@ describe('autocomplete', () => {
             cy.get(`#postMessageText_${postId}`).find('.mention-link.mention--highlight').should('exist');
         });
     });
-
-    it('MM-T2214 @ mention from link in profile popover: center', () => {
```

Comment:
This part is deleted because in the new design there is no option to mention user when clicking the username